aiida/orm/implementation/django/comments.py

```diff
@@ -116,3 +116,13 @@ def delete(self, comment):
             models.DbComment.objects.get(pk=comment).delete()
         except ObjectDoesNotExist:
             raise exceptions.NotExistent("Comment with id '{}' not found".format(comment))
+
+    # pylint: disable=no-self-use
+    def delete_many(self, filters):
+        """
+        Delete all comment entries in the table
+        """
+        if not filters:
+            models.DbComment.objects.all().delete()
```

Comment:
Maybe this should also be in a transaction

Reply:
How do I do this with Django?

Reply:
There is an example at the aiida.orm.importexport.import_data_dj
But maybe a transaction is not needed here. It is an one line statement/command and I hope that this is executed as one deletion sql statement (there is not reason not to). In such a case I think that the transaction will not help - sorry for the initial comment